SECURITY.md

```diff
@@ -0,0 +1,116 @@
+
+### Roles
+
+This process defines these roles:
+- **Reporter** The individual(s) who report the vulnerability
+- **Coordinator** A conda-forge core member who facilitates the tracking of the vulnerability
+  through this process
+- **Developer** One or more developers who work on remediating the vulnerability
+
+For the purpose of this document these roles are distinct, in practice, some of these roles may be handled by the same individual. However, the roles should be covered by a minimum of two separate individuals. For example, a Reporter may also fill the Developer role and create the remediation, in this case the Coordinator should be a separate individual.
+
+### Process
+
+The role responsible for each step is noted at the beginning.
+
+- Upon receipt of the initial report:
+  - **Coordinator**: Respond to the reported and acknowledge receipt of the report in the timeframe
+    given in the "Reporting Vulnerabilities" section.
+  - **Coordinator**: Open an issue in the private GitHub repository used for tracking
+    vulnerabilities across projects
+  - **Coordinator**: Review the issue for completeness and suitability (triage). If more
+    information is needed, follow up with the Reporter.
+- If the vulnerability is not accepted:
+  - **Coordinator**: Close the issue
+  - **Coordinator**: Notify the reporter
+- If the vulnerability is accepted, within the relevant repositories:
+  - **Coordinator**: Open a draft [GitHub Security
+    Advisory](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#about-github-security-advisories)
+    - Include relevant but sanitized details in the top level comment, which will become public
+    - Sensitive details and reproductions go in the comments on the draft advisory, which are not
+      public
+    - **Coordinator**: Add relevant people to the advisory
+    - **Developer**: Attempt to replicate the reported vulnerability. Request more information from
+      the **Reporter** if necessary.
+    - **Developer**: Work on the [vulnerability fix
+      PR](https://docs.github.com/en/code-security/repository-security-advisories/collaborating-in-a-temporary-private-fork-to-resolve-a-repository-security-vulnerability#creating-a-temporary-private-fork).
+    - **Coordinator**/**Developer**: If appliccable, request a
+      [CVE](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers)
+      from GitHub. The CVE Number will be private until the advisory is published.
+    - **Developer & Coordinator**: Decide on release and announcement dates and post them the draft
+      advisory.
+  - **Coordinator**: Post the release and announcement dates on the conda-forge core chat room and
+    mailing list.
+  - **Developer**: Merge the security fix PR
+  - **Developer**: Make a release to PyPI and/or npm with no announcement or change log
```

Comment:
This doesn't really apply or needs to be changed to be more general.

Reply:
True! Changed it to "release and/or deploy"

Reply:
True! Changed it to "release and/or deploy as needed"